SofascoreAcademy/SofascoreAcademy/Views/CustomTabView.swift

```diff
@@ -0,0 +1,121 @@
+import UIKit
+import SofaAcademic
+import SnapKit
+import Foundation
+
+class CustomTabView: BaseView {
+    
+    private let tabsContainerView = UIView()
+    private let tabWidth = UIScreen.main.bounds.width/3
```

Comment:
Vidim da si zaobisao problem razlicitih ekrana uredjaja i dobro si se snasao jer to radi, ali nije bas najbolji pristup. Jesi li probao ostvariti implementaciju da tabitemview-ove stavis u stackview, on bi taj layout dio odradio za tebe